middleware.ts

```diff
@@ -2,7 +2,27 @@ import collectionSlugToUuidMapping from "@/src/data/collectionSlugUuidMapping";
 import { deSlugify } from "@/src/utils/utils";
 import { NextRequest, NextResponse } from "next/server";
 
-export function middleware(req: NextRequest) {
+async function getCollectionTitle(uuid): Promise<string> {
+  try {
+    const response = await fetch(
```

Comment:
we talked about this and are in agreement that it's not ideal because this makes the middleware file async but is a faster solution to the problem than requesting an update to the collections filter. I'm ok with this for now